thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -330,16 +359,27 @@ export default Component.extend({
    * Checks if param values and displayed values differ by 5% or more
    * @type {Boolean}
    */
-  isWarning: computed('anomalyInfo', function () {
-    const anomalyInfo = get(this, 'anomalyInfo');
-    let oldCurrent = parseFloat(get(this, 'current'));
-    const newCurrent = this._getAggregate('current');
-
-    if (newCurrent && oldCurrent){
-      const diffCurrent = Math.abs((newCurrent-oldCurrent)/newCurrent);
-      return (diffCurrent > 0.01);
+  isWarning: computed('anomalyInfo', 'isRangeChanged', function () {
+    if(!get(this, 'isRangeChanged')) {
```

Comment:
Based on our discussion we will always show the alert regardless range change or not.

Reply:
Yes, this date range checker is in place because 'newCurrent' comes from _getAggregate, which will be updated if the user changes the date range.  So we make sure that the date range can't trigger the warning (In one case, the anomaly doesn't have the warning, but when the user slides the date picker, it activates the warning - 'isRangeChanged' is checked here to prevent this behavior).